tests/engines/registries/acceptance/branded/moderation/submissions-test.ts

```diff
@@ -244,4 +244,19 @@ module('Registries | Acceptance | branded.moderation | submissions', hooks => {
         assert.ok(currentURL().includes('state=embargo'), 'current URL contains the state query param set');
         assert.dom('[data-test-is-selected="true"]').hasText('Embargo', 'embargo tab selected');
     });
+
+    test('Submissions page: invalid queryParam', async function(this: ModerationSubmissionsTestContext, assert) {
+        this.registrationProvider.update({ permissions: ['view_submissions'] });
+        const currentUser = server.create('user', 'loggedIn');
+        server.create('moderator', {
+            id: currentUser.id,
+            user: currentUser,
+            provider: this.registrationProvider,
+        }, 'asModerator');
+        await visit('/registries/sbmit/moderation/submissions?state=embargooooo');
+        assert.equal(currentRouteName(), 'registries.branded.moderation.submissions',
+            'On the submissions page of registries reviews');
+        assert.notOk(currentURL().includes('state=embargooooo'), 'Invalid query param is gone');
```

Comment:
It'd be nice to another assert that checks the queryParam is `?state=pending`

Reply:
I agree that it would be best to have another assertion to check that the queryParam is reset, but I've noticed that the query param gets removed whenever `?state=pending`. I'm not sure why that is the case and I'd like to know what's causing this, but I will defer that exploration for now.

Reply:
Actually, found it in the documentation: https://guides.emberjs.com/release/routing/query-params/#toc_default-values-and-deserialization